deegree-ogcapi-features/src/test/java/org/deegree/services/oaf/resource/OpenApiVersionSegmentTest.java

```diff
@@ -0,0 +1,66 @@
+/*-
+ * #%L
+ * deegree-ogcapi-features - OGC API Features (OAF) implementation - Querying and modifying of geospatial data objects
+ * %%
+ * Copyright (C) 2019 - 2020 lat/lon GmbH, info@lat-lon.de, www.lat-lon.de
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Lesser Public License for more details.
+ *
+ * You should have received a copy of the GNU General Lesser Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ * #L%
+ */
+package org.deegree.services.oaf.resource;
+
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasJsonPath;
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.isJson;
+import static org.hamcrest.CoreMatchers.endsWith;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.deegree.services.oaf.OgcApiFeaturesMediaType;
+import org.deegree.services.oaf.filter.ApiVersionPathFilter;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.ProvideSystemProperty;
+
+/**
+ * Test OpenAPI resource with optional version segment enabled for path.
+ */
+public class OpenApiVersionSegmentTest extends OpenApiTest {
```

Comment:
In this file as well the indentation for the comments is shifted to the right when compared with the code.
![Screenshot 2023-03-31 at 16 14 59](https://user-images.githubusercontent.com/28185404/229145274-e34c5f90-75ca-4e81-a3c8-2a55ca829aa5.png)

Reply:
To me looks fine - maybe depending on the browser?
It would be good to have some common settings for the indentation and formatting. Ideally this would be something that is enforced/configured automatically. I would wait to see if there is any feedback on the official repo PRs before putting some effort into it.